src/Utils.js

```diff
@@ -1,4 +1,7 @@
 const url = require('url')
+// Removing utm_* parameters from URL
+// eslint-disable-next-line security/detect-unsafe-regex
+const REGEX_UTM_TRIMMER = /(\?)utm[^&]*(?:&utm[^&]*)*&(?=(?!utm[^\s&=]*=)[^\s&=]+=)|\?utm[^&]*(?:&utm[^&]*)*$|&utm[^&]*/gi
```

Comment:
@zivkaziv any chance we can do it without a regex?
one idea is perhaps to transform it to a structured URL, and then re-build it again to a URL string but excluding those query params that you remove here.

Reply:
Sure.. I'll use your suggestion:)

Reply:
Thanks! 
http://nodejs.org/api/url.html should set you up well